FINAL:
  prep_time: 5
  exec_time: 14
  objects: 10
  decoys: 8
  tables: 8
  obstacles: 2
  barrier_tapes: 2
  arbitrary_surfaces: 4
  pp_placing: 2
  container_placing_r: 2
  container_placing_b: 2
  paired_containers: false
  tt_grasping: 2
  tt_placing: 0
  ref_tt_direction: true
  shelfes_grasping: 2
  shelfes_placing:  2
  ref_position: true
  ref_rotation: true
  ref_orientation: true
  ref_cavity_position: true
  ref_cavity_rotation: true
  ref_cavity_orientation: true
